app/src/main/java/io/intrepid/contest/models/Entry.java

```diff
@@ -0,0 +1,12 @@
+package io.intrepid.contest.models;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class Entry {
+    public UUID id;
+    public String title;
+    public Date createdAt;
+    public Date updatedAt;
+    public String photoUrl;
```

Comment:
we arent using createdAt, updatedAt, and photourl, these are part of the object we get back though right? I think I asked that already. If so then leave it.